spec/features/case_information_feature_spec.rb

```diff
@@ -84,13 +84,15 @@
     it "clicking back link after viewing prisoner's case information, returns back the same paginated page",
        vcr: { cassette_name: 'prison_api/case_information_back_link' }, js: true do
       visit missing_information_prison_prisoners_path('LEI', page: 3)
+      expect(current_page_number).to eq(3)
```

Comment:
@ollietreend Why is checking the page number important?

Reply:
@Katy600 the test scenario is:

> clicking back link after viewing prisoner's case information, **returns back the same paginated page**

But the test was only asserting that we end up on a page with the heading 'Add missing details'. It didn't actually prove that we returned to page 3 of the listings, so wasn't testing what it claimed to test. We could have actually returned to page 1 of the results and the test would still have passed.

That's why I've added the check for page numbers. Just to make it clear that it is, in fact, behaving correctly.

Reply:
Oh of course! Thanks @ollietreend! That makes so much sense :)